### ISSUE TITLE
download link is broken

### ISSUE THREAD

**Issue by davideghz** [rafaelp/heroku-buildpack-wkhtmltopdf#10]
http://download.gna.org/wkhtmltopdf/0.12/0.12.2.1/wkhtmltox-0.12.2.1_linux-trusty-amd64.deb
is no longer active, can you pls update your buildpack with a new link?

**Comment by OGKevin** [comment]
@davideghz have you found a fix for this yet ?

**Comment by davideghz** [comment]
yes, here https://github.com/dscout/wkhtmltopdf-buildpack/pull/19
